lib/capistrano/fiesta/report.rb

```diff
@@ -70,20 +70,33 @@ def stories_with_release_notes
         end
 
         def merged_pull_requests
-          github.search_issues("base:master repo:#{repo} merged:>#{last_released_at}").items
-        rescue Octokit::UnprocessableEntity => e
-          Logger.warn "Unable to access GitHub. Message given was: #{e.message}"
-          []
+          merged_pull_request_numbers.map do |number|
+            begin
+              github.pull_request(repo, number)
+            rescue Octokit::NotFound
+              nil
+            end
+          end.compact
         end
 
-        def repo
-          github_url.match(/github.com[:\/](\S+\/\S+)\.git/)[1]
+        def merged_pull_request_numbers
+          commits.map do |commit|
+            commit.commit.message.slice(/\AMerge pull request #(\d+) /, 1)
```

Comment:
I'm not sure if I follow this correctly, but does this rely on merge commits? If so, we often use squash or rebase, so the commit messages might not follow this pattern...?